modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -780,15 +780,15 @@ class CV_EXPORTS TrackerSamplerCSC : public TrackerSamplerAlgorithm
     MODE_DETECT = 5   //!< mode for detect samples
   };
 
-  struct CV_EXPORTS Params
+  struct CV_EXPORTS_W_SIMPLE Params
   {
-    Params();
-    float initInRad;        //!< radius for gathering positive instances during init
-    float trackInPosRad;    //!< radius for gathering positive instances during tracking
-    float searchWinSize;  //!< size of search window
-    int initMaxNegNum;      //!< # negative samples to use during init
-    int trackMaxPosNum;     //!< # positive samples to use during training
-    int trackMaxNegNum;     //!< # negative samples to use during training
+    CV_WRAP Params();
```

Comment:
> CV_WRAP

Remove from this line.

---

Similar is below.

Reply:
@alalek thank you. It works. We can still use default constructor in Python.